lib/db/file_handler.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+class FileHandler
+
+  def initialize(file)
+    @file = file
+  end
+
+  def read
```

Comment:
The method name is `read`, but you're actually parsing what is read from the file. Can you think of a better name?